src/main/java/com/beerjournal/infrastructure/error/RestResponseExceptionHandler.java

```diff
@@ -21,6 +21,6 @@
         if (ex instanceof BeerJournalException) {
             return handleBeerJournalException((BeerJournalException) ex);
         }
-        return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+        return new ResponseEntity<>(ex.getMessage(), HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
Sure, it's not a good idea to expose this to the client, basically I got 500 a couple of times and needed to know what's going one and later forgot to edit this out :P. But still, I think we should log (when we add logger xD) this message on the server because otherwise we have no information from the server other than 500 error code. Will change this later.